src/constants/notificationSchema.js

```diff
@@ -2,7 +2,7 @@ export const notificationTypeNames = {
   collaboration_manager_create: 'collaboration_manager_create',
   collaboration_request: 'collaboration_request',
   collaboration_approved: 'collaboration_approved',
-  collaboration_revoke: 'collaboration_revoke',
+  collaboration_revoke: 'collaboration_denied',
```

Comment:
Are you sure about this? Both of them are houston Notification types
https://github.com/WildMeOrg/houston/blob/923dbab77fc1bb1c6105ec7d2b32bb7860c9f78b/app/modules/notifications/models.py#L18

Reply:
Good catch. Both should exist. Should be implemented now.